portal/models/fhir.py

```diff
@@ -519,14 +529,12 @@ def consolidate_answer_pairs(answers):
         answer_types = [a.keys()[0] for a in answers]
 
         # Exit early if assumptions not met
-        if (
-            len(answers) % 2 or
-            answer_types.count('valueCoding') != answer_types.count('valueString')
-        ):
+        if (len(answers) % 2 or answer_types.count('valueCoding')
```

Comment:
Please don't change the other lines unless necessary; ie keep the original, but break the long line on `!=`